packages/robotics/pitop/robotics/drive_controller.py

```diff
@@ -79,8 +78,13 @@ def _set_synchronous_motor_movement_mode(self) -> None:
         )
         self.__mcu_device.write_byte(MotorSyncRegisters.CONFIG.value, sync_config)
 
+    def _unset_synchronous_motor_movement_mode(self) -> None:
+        self.__mcu_device.write_byte(MotorSyncRegisters.CONFIG.value, 0b0000000)
+
     def _start_synchronous_motor_movement(self) -> None:
+        self._set_synchronous_motor_movement_mode()
```

Comment:
instead of doing this sync mode either side of the 'start' we need to do it either side of setting the target rpms and start, eg doing it and the start and end of  _set_motor_speeds(